specification/resources/resource-manager/Microsoft.Authorization/stable/2019-06-01/policyAssignments.json

```diff
@@ -0,0 +1,716 @@
+        "x-ms-examples": {
+          "Retrieve a policy assignment by ID": {
+            "$ref": "./examples/getPolicyAssignmentById.json"
+          },
+          "Retrieve a policy assignment with a managed identity by ID": {
+            "$ref": "./examples/getPolicyAssignmentWithIdentityById.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "policyAssignmentId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the policy assignment to get. Use the format '{scope}/providers/Microsoft.Authorization/policyAssignments/{policyAssignmentName}'.",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the policy assignment.",
+            "schema": {
+              "$ref": "#/definitions/PolicyAssignment"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PolicyAssignmentProperties": {
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the policy assignment."
+        },
+        "policyDefinitionId": {
+          "type": "string",
+          "description": "The ID of the policy definition or policy set definition being assigned."
+        },
+        "scope": {
+          "type": "string",
+          "description": "The scope for the policy assignment."
+        },
+        "notScopes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "The policy's excluded scopes."
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Required if a parameter is used in policy rule."
+        },
+        "description": {
+          "type": "string",
+          "description": "This message will be part of response in case of policy violation."
+        },
+        "metadata": {
+          "type": "object",
+          "description": "The policy assignment metadata."
+        },
+		"enforcementMode": {
+          "type": "string",
+          "description": "The policy assignment enforcement mode. Possible values are NotSpecified, Default, and DoNotEnforce.",
+          "enum": [
+            "NotSpecified",
+            "Default",
```

Comment:
>Default [](start = 13, length = 7)

Default is the same as NotSpecified. Do we really need both of them?

Reply:
NotSpecified should be an internal value, not exposed to users.  You default to "Default" if the user did not provide a value don't you?

---
In reply to: [316790254](https://github.com/Azure/azure-rest-api-specs/pull/6998#discussion_r316790254) [](ancestors = 316790254)

Reply:
Makes sense. If we make sure that we never return that from the service, then we should probably remove it from the Swagger here.

Reply:
Although I think I'm ok if we keep it or remove it.